### DISCUSSION TITLE
SHTC3 versions

### DISCUSSION THREAD

**Discussion by breakthastatic** [rbaron/b-parasite#222] [Q&A]
I'm working on getting an order together with JLCPCB and they do not have enough of the part SHTC3-TR-2.5KS in stock.  They do have a large stock of SHTC3-TR-10KS chips though.  I tried comparing the datasheets which are different, but I can't spot anything significantly different.  Does anyone know if the SHTC3-TR-10KS will work as a drop in replacement?

**Comment by rbaron** [comment]
Google pointed me to https://forum.digikey.com/t/difference-of-sensirion-shtc3-tr-10ks-vs-shtc3-tr-2-5ks-with-full-tape-and-reel/24722

According to the post both parts are the same.

**Reply by breakthastatic** [reply]
Thanks!  My google skills let me down there.
